generators/Exercises/Hamming.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using Generators.Input;
+
+namespace Generators.Exercises
+{
+    public class Hamming : Exercise
+    {
+        protected override void UpdateCanonicalData(CanonicalData canonicalData)
+        {
+            foreach (var canonicalDataCase in CanonicalData.Cases)
+            {
+                canonicalDataCase.ExceptionThrown = canonicalDataCase.Expected is long ? null : typeof(ArgumentException);
+            }
```

Comment:
At the moment, I think just using `long` is sufficient, but once it gets problematic we can change things.